build.gradle

```diff
@@ -769,7 +769,7 @@ task downloadJtreg(type: Download) {
     // dest new File(buildDir, 'jtreg-4.2.0-tip.tar.gz')
     // src 'https://builds.shipilev.net/jtreg/jtreg4.2-b16.zip'
     src 'https://builds.shipilev.net/jtreg/jtreg-7.5+1.zip'
-    dest new File(buildDir, 'jtreg.zip')
+    dest new File(layout.buildDirectory.asFile.get(), 'jtreg.zip')
```

Comment:
What's the difference between `asFile.get()` and `get().asFile` used elsewhere?

Reply:
Wouldn't `layout.buildDirectory.file('jtreg.zip').get().asFile` be more idiomatic?

Reply:
> What's the difference between asFile.get() and get().asFile used elsewhere?

That's a good question. I will ask gradle people on slack. Looks like both of them return a file.